release-notes/SupportedFeatures-v1.0.0-rc1.md

```diff
@@ -28,9 +28,9 @@ Supported features
 ||NetHttpBinding|:white_check_mark: | :white_check_mark: | :white_check_mark: | :warning: [more...](https://github.com/dotnet/wcf/issues/534) |
 ||NetTcpBinding|:white_check_mark: | :white_check_mark: | :white_check_mark: | :warning: [more...](https://github.com/dotnet/wcf/issues/534) |
```

Comment:
We could make "more..." a query, but I couldn't find any clean way to form accurate queries unique to the feature for that row in the table.  I think it might be reasonable to make the NET Native and Linux columns use a query for labels "bug" and "NET Native" (or "Linux").  What do you think of adding new labels that correspond to the cells in the "Feature" column?  It would allow self-maintaining queries and give us a better way to indicate "what feature is affected by this bug?".